Project2_2015.py

```diff
@@ -1,40 +1,14 @@
-box_dimensions = [0, 0, 0]
-box_dimensions_area = [0, 0, 0]
-box_dimensions_perimeter = 0
-box_surface_area = 0
-box_wrapper_sq = 0
 total_wrapper = 0
-box_ribbon = 0
 total_ribbon = 0
-current_number = ''
 
 with open('Project2_2015', 'r') as file:
-    for index, lines in enumerate(file):
-        current_line = lines
-        side_tracker = 0
-        for char_index, char in enumerate(current_line):
-            if char == 'x':
-                box_dimensions[side_tracker] = int(current_number)
-                side_tracker += 1
-                current_number = ''
-            elif char_index + 1 == len(current_line):
-                if current_number == '':
-                    current_number += char
-                box_dimensions[side_tracker] = int(current_number)
-                current_number = ''
-                box_dimensions_area[0] = box_dimensions[0] * box_dimensions[1]
-                box_dimensions_area[1] = box_dimensions[1] * box_dimensions[2]
-                box_dimensions_area[2] = box_dimensions[2] * box_dimensions[0]
-                box_dimensions.sort()
-                box_dimensions_perimeter = 2*(box_dimensions[0] + box_dimensions[1])
-                box_ribbon = box_dimensions_perimeter + (box_dimensions[0]*box_dimensions[1]*box_dimensions[2])
-                box_dimensions_area.sort()
-                box_surface_area = 2 * (box_dimensions_area[0] + box_dimensions_area[1] + box_dimensions_area[2])
-                box_wrapper_sq = box_surface_area + box_dimensions_area[0]
-                total_wrapper += box_wrapper_sq
-                total_ribbon += box_ribbon
-            else:
-                current_number += char
+    for line in file:
+        l, w, h = sorted([int(i) for i in line.split('x')])
```

Comment:
in retrospect these could be called anything and in any order (however you want to perceive a 3d object where largest dimensions come first); e.g. `width, height, depth` might be more natural and better naming than `w,  h, d` (or `l, w, h` as it is currently)

Reply:
also obviously they don't need to be destructured but saying `dimensions[0]` all the time gets old (IMO) 😄

Reply:
`d0, d1, d2`? 🤷

Reply:
so I took one CS course in college and the professor was anal about names. He hated abbreviations, because others would have a hard time reading it. In your experience, what is typically used, full names or abbreviations?

Reply:
full names except for let maybe like `i` for `index` in for loops (but really that type of code is generally avoided for things like `for line in file`, etc.